src/Mvc/Mvc.Core/src/ControllerBase.cs

```diff
@@ -1858,9 +1909,44 @@ public virtual ActionResult ValidationProblem([ActionResultObjectValue] ModelSta
         /// </summary>
         /// <returns>The created <see cref="BadRequestObjectResult"/> for the response.</returns>
         [NonAction]
-        public virtual ActionResult ValidationProblem()
+        public virtual ActionResult ValidationProblem() => ValidationProblem(ModelState);
+
+        /// <summary>
+        /// Creates an <see cref="BadRequestObjectResult"/> that produces a <see cref="StatusCodes.Status400BadRequest"/> response
+        /// with a <see cref="ValidationProblemDetails"/> value.
+        /// </summary>
+        /// <param name="title">The value for <see cref="ProblemDetails.Title" />.</param>
+        /// <param name="type">The value for <see cref="ProblemDetails.Type" />.</param>
+        /// <param name="detail">The value for <see cref="ProblemDetails.Detail" />.</param>
+        /// <param name="instance">The value for <see cref="ProblemDetails.Instance" />.</param>
+        /// <param name="modelStateDictionary">The <see cref="ModelStateDictionary"/>. 
+        /// When <see langword="null"/> uses <see cref="ModelState"/>.</param>
+        /// <returns>The created <see cref="BadRequestObjectResult"/> for the response.</returns>
+        [NonAction]
+        public virtual ActionResult ValidationProblem(
+            string detail,
+            string instance = null,
+            string title = null,
+            string type = null,
+            [ActionResultObjectValue] ModelStateDictionary modelStateDictionary = null)
         {
-            var validationProblem = new ValidationProblemDetails(ModelState);
+            modelStateDictionary ??= ModelState;
+
+            var validationProblem = new ValidationProblemDetails(modelStateDictionary)
+            {
+                Detail = detail,
+                Instance = instance,
+                Type = type,
+            };
+
+            if (title != null)
+            {
+                // ValidationProblemDetails has a Title by default. Do not overwrite it with a null
+                validationProblem.Title = title;
+            }
+
+            ApplyProblemDetailsDefaults(validationProblem, statusCode: 400);
```

Comment:
For the people that really want this to be a 422 - what do they do?

Reply:
`ValidationProblem` returns the non-specific `ActionResult` return type. We expect you to override and copy this block of code if you'd like it to get this behavior. It's a few lines of code to copy.